lib/bezier.js

```diff
@@ -718,7 +718,7 @@
         return utils.between(p.x, mx, MX) && utils.between(p.y, my, MY);
       });
     },
-    selfintersects: function() {
+    selfintersects: function(threshold) {
```

Comment:
given the two different names, it might make sense to rename this arg to `curveIntersectionThreshold` as well